tests/reftests/env.win32.test

```diff
@@ -85,13 +137,14 @@ setenv: [
   [ RCTQ_ENVSET_ADD_WITH_COL += "a:/nother/gi;ven/path" ]
 ]
 build-env: [
+  [ LC_ALL = "C" ]
```

Comment:
I feel like we'd want this all the time, like globally. Couldn't we add this in `run.ml` instead?

Reply:
I debated this, too - but the problem then is that it's really not visible from the test that we've done that, and I anticipated that being confusing in future... especially as it would then be really unclear if it ever got changed or broken how and when it might then fail.